src/Symfony/Component/Console/Output/ConsoleSectionOutput.php

```diff
@@ -55,11 +55,14 @@ public function clear(int $lines = null)
             return;
         }
 
-        if (!$lines) {
+        if ($lines) {
+            $newContent = implode(PHP_EOL, \array_slice(explode(PHP_EOL, $this->content), 0, -$lines - 1));
```

Comment:
@nicolas-grekas Can you please give advice if there is easy way to optimize this line?

Reply:
Not sure :)

Reply:
Ok thanks, just wanted to know if there isn't something obvious by the chance